docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -116,6 +116,37 @@ for data that matches the query.
 |===
 // end::security-cloudtrail-jobs[]
 
+[discrete]
+[[security-host-jobs]]
+== Security: Host
+
+Anomaly detection jobs for host-based threat hunting and detection.
+
+In the {ml-app} app, these configurations are available only when data exists
+that matches the query specified in the
+https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_host/manifest.json[manifest file].
+In the {security-app}, it looks in the {data-source} specified in the
+{kibana-ref}/advanced-options.html#securitysolution-defaultindex[`securitySolution:defaultIndex` advanced setting]
+for data that matches the query.
+
+// tag::security-host-jobs[]
+
+|===
+|Name |Description |Job |Datafeed
+
+|high_count_events_for_a_host_name
+|Looks for a sudden spike in host based traffic. This can be due to a range of security issues, such as a compromised system, DDoS attacks, malware infections, privilege escalation, or data exfiltration.
+|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_host/ml/high_count_events_for_a_host_name.json[image:images/link.svg[A link icon]]
+|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_host/ml/datafeed_high_count_events_for_a_host_name.json[image:images/link.svg[A link icon]]
+
+|low_count_events_for_a_host_name
+|Looks for a sudden drop in host based traffic. This can be due to a range of security issues, such as a compromised system, a failed service, or a network misconfiguration.
+|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_host/ml/low_count_events_for_a_host_name.json[image:images/link.svg[A link icon]]
```

Comment:
The preview looks good, but the links to GitHub are broken:

https://stack-docs_bk_2909.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-ml-jobs.html#security-host-jobs
<img width="1661" alt="Screenshot 2025-01-21 at 12 21 42 PM" src="https://github.com/user-attachments/assets/975361e0-d6fc-4b6b-b097-b058810cfb69" />
<img width="1660" alt="Screenshot 2025-01-21 at 12 23 36 PM" src="https://github.com/user-attachments/assets/9f6881de-d235-4041-9ffa-58678524d281" />

Reply:
The links are broken for other modules as well. I believe this might be because the branch here points to 8.18 or a similar version.

Reply:
True, very odd yeah! In the preview site, when I selected 8.17 the links worked. So I guess we can take it with a grain of salt - but let's check back when it's merged/deployed